mutating-webhook/webhook.go

```diff
@@ -362,8 +362,7 @@ func handleDelete(ar *v1.AdmissionReview) *v1.AdmissionResponse {
 		}
 	}
 
-	if req.Kind.Kind == "Namespace" {
-
+	if kind == "Namespace" {
```

Comment:
Good simplification. Earlier we are already setting and using kind, so makes sense to use it here.